ports/mimxrt/modules/_boot.py

```diff
@@ -3,13 +3,31 @@
 # Note: the flash requires the programming size to be aligned to 256 bytes.
 
 import os
+import sys
 import mimxrt
+from machine import SDCard
```

Comment:
this will fail if `machine` doesn't have `SDCard`

Reply:
Fixed